extensions/schema-compare/src/test/schemaCompareDialog.test.ts

```diff
@@ -66,4 +73,28 @@ describe('SchemaCompareDialog.openDialog @DacFx@', function (): void {
 			applyButtonState: false
 		});
 	});
+
+	it('Verify database dropdown gets populated appropriately', async function (): Promise<void> {
```

Comment:
should this be server dropdown? Looks like only the server dropdown values are getting checked after connecting

Reply:
Yes, it is server dropdown and not database. Updated it accordingly.